data/transition-sites/mod_sabre.yml

```diff
@@ -0,0 +1,8 @@
+---
+site: mod_sabre
+whitehall_slug: ministry-of-defence
+homepage: https://www.gov.uk/government/organisations/ministry-of-defence
+host: www.sabre.mod.uk
+tna_timestamp: 20150806090301
```

Comment:
This confused me a bit: the latest crawl [listed on TNA](http://webarchive.nationalarchives.gov.uk/*/http://www.sabre.mod.uk) is `20150417170249`, but we've had confirmation that `20150806090301` will be the timestamp for a crawl which has recently been completed once it's put live. At the moment content from the earlier crawl is [displayed at the URLs](http://webarchive.nationalarchives.gov.uk/20150806090301/http://www.sabre.mod.uk/) for the recent one.